java/jsinterop/generator/closure/visitor/MemberCollector.java

```diff
@@ -123,13 +123,17 @@ protected void endVisitConstructor(FunctionType constructor) {
 
   @Override
   protected boolean visitField(StaticTypedSlot<JSType> jsField, boolean isStatic) {
-    getCurrentJavaType()
-        .addField(
-            Field.create(
-                extractName(jsField.getName()),
-                getJavaTypeRegistry().createTypeReference(jsField.getType()),
-                isConstant(jsField),
-                isStatic));
+    Field field = Field.create(
+            extractName(jsField.getName()),
+            getJavaTypeRegistry().createTypeReference(jsField.getType()),
+            isConstant(jsField),
+            isStatic);
+
+    if(jsField.getJSDocInfo()!=null && jsField.getJSDocInfo().isDeprecated()) {
```

Comment:
whitespace nit: space between `if` and `(`, and surrounding `!=` (next block of code too

Reply:
fixed